lua/luaGlobal.h

```diff
@@ -41,11 +42,37 @@ static int hslua_g_printMessage(lua_State* lua)
     return 0;
 }
 
+static int hslua_g_printToScreen(lua_State* lua)
+{
+    std::string message = std::string();
+    int argCount = lua_gettop(lua);
+    lua_getglobal(lua, "tostring");
+    for (int i = 1; i <= argCount; i++)
+    {
+        lua_pushvalue(lua, -1);
+        lua_pushvalue(lua, i);
+        lua_call(lua, 1, 1);
+        const char* str = lua_tostring(lua, -1);
+        if (str == NULL)
+        {
+            return luaL_error(lua, "'tostring' must return a string to 'print'");
+        }
+        
+        message += str;
+        message += "    ";
+        lua_pop(lua, 1);
+    }
+    PrintHelper::GetInstance()->AddMessage(message);
+    printf("%s\n", message.c_str());
```

Comment:
Lets also call
```c++
hs_log_file("%s", logMessage.c_str());
```
So that it's logged to both.
Lets also please format the message with `[Lua]: %s\n` like the log message does so it's clear which logs are from lua and which are from other HS internals.

Reply:
Should I just do that for `printf` and `hs_log_file` because those will be used by HS internals? Orr should I do it for the message used by `PrintHelper` as well? I'm inclined to do it for all three  because it is possible that `PrintHelper` would be used by some HS internals in the future, such as issue #128.

Reply:
Well `PrintHelper` displays on screen right?
Humm... #128 is a good point though...

Actually no, don't do it for `PrintHelper` because this is used to show messages to the user, the other is really more for us having a log for debugging and knowing "oh those lines came from Lua, etc..." and not from something else within HS.

If they're logging a error-case they can just use `log(` to prevent it from displaying on-screen to the user.

Reply:
Made that change, also added varargs/non-string arg support for `log`